shared/src/types/admin/Payment.ts

```diff
@@ -1,11 +1,29 @@
+import { Timestamp } from '@google-cloud/firestore';
+
 export const PAYMENT_FIRESTORE_PATH = 'payments';
 
+export enum PaymentStatus {
+	Created = 'created',
+	Paid = 'paid',
+	Confirmed = 'confirmed',
+	Contested = 'contested',
+	Failed = 'failed',
+	Other = 'other',
+}
+
 export type Payment = {
```

Comment:
@MDikkii: I slightly updated the fields on the payment document. You might have to update this on your end as well (payment status values are a little different, some fields are removed, and a `comments` field was added).

Reply:
@mkue what does that mean for the existing values (e.g. payment status) in firestore? Will they be displayed correctly in the admin?

Reply:
yes, all payments are currently set to 'confirmed', so it shouldn't be a problem at all.

Reply:
ok, there are 10 entries that need to be updated. I'll do it by hand quickly

Reply:
Don't we want to keep some timestamp for changes (confirmation / contest timestamp)? 

Contest reason probably can go to `comments`, right?

Reply:
Yes, I would say so. The changes are now automatically logged to a subcollection, see #323. So the information is there, and if needed for some reason, we can also add the fields back.